test/features/cloudnative/network_zones/name.go

```diff
@@ -0,0 +1,31 @@
+package network_zones
+
+import (
+	"crypto/rand"
+)
+
+const prefix = "op-e2e-"
+const defaultName = "test-network-zone"
+
+// Use a randomized network zone names, to avoid problems with improper or slow cleanup of network zones on DT cluster side.
+// With randomized names a fresh start is guaranteed for each test run.
+func getNetworkZoneName() string {
```

Comment:
small nitpick: i was wondering if we need something similar in another tests, wdyt to move this code to helpers, and import there like getRandomBytes(length int). and reuse inside getNetworkZoneNames?